ci/scripts/deterministic-recovery-test.sh

```diff
@@ -44,7 +44,7 @@ filter_stack_trace_for_all_logs() {
   done
 }
 
-trap filter_stack_trace_for_all_logs ERR
+# trap filter_stack_trace_for_all_logs ERR
```

Comment:
We can just remove the log filter, until recovery tests stabilize. The recent logs are < 1MB, and we only upload the zipped archive for the logs.